string object_name
float32 x
float32 y
float32 angle
float32 length
float32 width
int32 yolo_bbox_center_x
int32 yolo_bbox_center_y
int32 yolo_bbox_min_x
int32 yolo_bbox_max_x
int32 yolo_bbox_min_y
int32 yolo_bbox_max_y
int32 yolo_bbox_s
